### PR TITLE
Fixed access modifiers for inner classes to avoid classloading issues

### PR DESCRIPTION
**Pull request by romartin**
As discussed with Pete, package protected access modifiers are causing class loading issues.

### CODE REVIEW

File: impl/src/main/java/org/jboss/solder/servlet/event/ImplicitServletObjectsHolder.java [hunk 1]
```diff
@@ -161,10 +161,13 @@ public HttpSession getHttpSession() {
         }
     }
 
-    static class InternalServletContextEvent {
+    /**
```
Comment (pmuir) [inline impl/src/main/java/org/jboss/solder/servlet/event/ImplicitServletObjectsHolder.java]: I'm not sure this should be a javadoc comment, just a normal comment...

Comment (romartin) [inline impl/src/main/java/org/jboss/solder/servlet/event/ImplicitServletObjectsHolder.java]: Sorry Pete, you are right.. this should not be a javadoc comment, just a normal one... Do you want me to do another pull request fixing that comments or can you change these comments as non-javadoc when pushing? 
Thx

Comment (pmuir) [inline impl/src/main/java/org/jboss/solder/servlet/event/ImplicitServletObjectsHolder.java]: Just push the changes to your branch, then the pull request will update.

Comment (romartin) [inline impl/src/main/java/org/jboss/solder/servlet/event/ImplicitServletObjectsHolder.java]: Done!

Thanks

On Wed, Aug 7, 2013 at 3:05 PM, Pete Muir notifications@github.com wrote:

> In
> impl/src/main/java/org/jboss/solder/servlet/event/ImplicitServletObjectsHolder.java:
> 
> > @@ -161,10 +161,13 @@ public HttpSession getHttpSession() {
> >          }
> >      }
> > -    static class InternalServletContextEvent {
> > -    /**
> 
> Just push the changes to your branch, then the pull request will update.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/seam/solder/pull/81/files#r5630541
> .

## 

Roger Martínez
roger600@gmail.com

**Comment by pmuir** [comment]
Merged
